shared/src/inclusionConnect/inclusionConnect.dto.ts

```diff
@@ -23,10 +24,16 @@ export type AuthenticatedUserQueryParams = {
   token: string;
 } & Pick<AuthenticatedUser, "email" | "firstName" | "lastName">;
 
-export const getUserRoleForAccessingConvention = (
-  convention: ConventionReadDto,
+export type InclusionConnectConventionManageAllowedRoles =
+  | ExcludeFromExisting<AgencyRole, "toReview">
+  | Extract<SignatoryRole, "establishment-representative">;
```

Comment:
Pour assurer le typage on peut faire :
```suggestion
  | ExtractFromExisting<SignatoryRole, "establishment-representative">;
```